README-agile.md

```diff
@@ -0,0 +1,145 @@
+# AGILE scripts
+
+This repository contains start and stop scripts for various [AGILE](http://agile-iot.eu/) components.
+Currently, AGILE is tested on Raspberry PI 2/3 Model B running Raspbian, and on x86_64 running Ubuntu 16.04.
+Since we use docker containers, it should also run in many other environment.
+
+## Requirements
+
+### On Raspberry PI 2/3 Model B running Raspbian
```

Comment:
re: keeping hypriot instructions. We could but it adds another path we have to support/troubleshoot. I think we should narrow options if it makes sense.